.yarn/patches/@metamask-network-controller-npm-18.1.0-680908c29a.patch

```diff
@@ -1,23 +1,23 @@
-diff --git a/dist/chunk-VNCJZRDU.js b/dist/chunk-VNCJZRDU.js
```

Comment:
Making sure this patch change was intended. I'm not actually clear on how these patches are generated and maintained

Reply:
Yes this was intended. Its updated to point at the latest networkController version which we want to bump to here. It is required to pass [this recently added test](https://github.com/MetaMask/metamask-extension/blob/eb93c9847cbb0769a260d53e70262f32a51e9cb9/test/e2e/tests/onboarding/onboarding.spec.js#L311) (for which the patch was added)